apps/web/components/booking/AvailableEventLocations.tsx

```diff
@@ -23,8 +26,8 @@ export function AvailableEventLocations({ locations }: { locations: Props["event
               )}
               alt={`${eventLocationType.label} icon`}
             />
-            <Tooltip content={locationKeyToString(location)}>
-              <p className="truncate">{locationKeyToString(location)}</p>
+            <Tooltip content={t(locationKeyToString(location) ?? "")}>
+              <p className="truncate">{t(locationKeyToString(location) ?? "")}</p>
```

Comment:
what does ?? "" do? havent seen that before with t() function

Reply:
It serves as a fallback, if `locationKeyToString` returns`undefined`.